Libraries/Starlight.Rpc/Protobuf/starlight.player.proto

```
@@ -46,6 +78,19 @@ message FetchPlayerRsp {
 
+// Path: Game Server -> DB Gate
+// Subject: `game.player.save`
+message SavePlayerReq {
+  uint32 uid = 1;
+  NetPlayerState state = 2;
+}
+
+// Path: DB Gate -> Game Server
+// Subject: `game.player.save`
+message SavePlayerRsp {
+  StarlightRetcode retcode = 1;
```

Buf (1.72.0)

[error] cannot find `StarlightRetcode` in this scope

(COMPILE)